src/LoadingSkeleton/LoadingSkeleton.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Skeleton, { SkeletonTheme } from 'react-loading-skeleton';
+import 'react-loading-skeleton/dist/skeleton.css';
+
+import colors from '../Styles/colors/palette';
+
+const LoadingSkeleton = ({
+ borderRadius,
+ circle,
+ className,
+ containerClassName,
+ containerTestId,
+ count,
+ height,
+ inline,
+ width,
+ ...props
+}) => (
+  <SkeletonTheme baseColor={colors.UX_GRAY_300}>
+    <Skeleton
+      borderRadius={borderRadius}
+      circle={circle}
+      className={classNames('LoadingSkeleton', className)}
+      containerClassName={containerClassName}
+      containerTestId={containerTestId}
+      count={count}
+      height={height}
+      inline={inline}
+      width={width}
+      {...props}
+    />
+  </SkeletonTheme>
+);
+
+LoadingSkeleton.propTypes = {
+  /**
+  The border radius of the skeleton.
+  */
+  borderRadius: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+  Makes the skeleton circular by setting border-radius to 50%.
+  */
+  circle: PropTypes.bool,
+  className: PropTypes.string,
+  /**
+  A custom class name for the <span> that wraps the individual skeleton elements.
+  */
+  containerClassName: PropTypes.string,
+  /**
+  A string that is added to the container element as a data-testid attribute.
+  Use it with screen.getByTestId('...') from React Testing Library.
+  */
+  containerTestId: PropTypes.string,
+  /**
+  The number of lines of skeletons to render. If count is a decimal number like 3.5,
+  three full skeletons and one half-width skeleton will be rendered.
+  */
+  count: PropTypes.number,
+  /**
+  The height of the skeleton.
+  */
+  height: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+  By default, a <br /> is inserted after each skeleton so that each skeleton gets its own line.
```

Comment:
💅 Similar issue here:

<img width="691" alt="Screen Shot 2023-01-04 at 4 11 32 PM" src="https://user-images.githubusercontent.com/3161151/210650732-ae023181-c170-4be6-8a70-f92ff5c8fe1d.png">

```suggestion
  By default, a `<br />` is inserted after each skeleton so that each skeleton gets its own line.
```